README.md

```diff
@@ -1,124 +1,89 @@
 # dojot History service
```

Comment:
The title is about the history service, but the content is about the `history` and `persister`. Isn't clear what is the role of each one.

You could modify the document structure in order to improve the understanding. I suggest something like:
A brief description talking about the repository itself.
A section exclusively to the `history`
A section exclusively to the `persister`

The service's section with:
- Description
- Configuration
- Environment variables
- Installation (stand-alone and container case)
- How to run (stand-alone and container case)
- API

Remember: the services are together, but they are not the same service.